packages/components/src/AsyncComponent/index.tsx

```diff
@@ -49,7 +48,7 @@ const BaseAsyncComponent: React.FunctionComponent<BaseAsyncComponentProps> = ({
     ...props,
   };
   return (
-    <Cmp className={classNames(className, appName)}>
+    <Cmp className={classNames(className, appName, 'pf-v5-u-h-100')}>
```

Comment:
This is not something we can. You can have multiple async components on a page and this will affect the overall layout of a page. You need to do this in your loading components that required the extra styling.